pkg/apis/hub/v1alpha1/api_portal.go

```diff
@@ -23,12 +23,10 @@ import (
 )
 
 // +genclient
-// +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // APIPortal defines a portal that exposes APIs.
```

Comment:
Not related to your PR it's a good opportunity to improve this comment.
```suggestion
// APIPortal defines a developer portal for accessing the documentation of APIs.
```